lib/gooddata/lcm/actions/collect_dynamic_schedule_params.rb

```diff
@@ -57,8 +57,15 @@ def call(params)
             GoodData.logger.debug("Processing row: #{safe_to_print_row}")
             results << safe_to_print_row
 
-            client_id = row[client_id_column] || :all_clients
-            schedule_name = row[schedule_title_column] || :all_schedules
+            fail "Column #{client_id_column} doesn't exist in the input." unless row.key?(client_id_column)
+
+            client_id_column_value = row[client_id_column]
+            client_id = client_id_column_value.blank? ? :all_clients : client_id_column_value
+
+            fail "Column #{schedule_title_column} doesn't exist in the input." unless row.key?(schedule_title_column)
```

Comment:
move both checks of missing columns of input file (csv) outside the main loop / check header before main loop only ( if it is not checked elsewhere )

Reply:
Done remove source code checks of missing columns of input file. We don't need the checking for them. The Provision or Rollout bricks should not failed by the missing columns. In case missing columns, the behavior will same as existing the columns with empty value.